spec/dummy/app/avo/actions/test/close_modal.rb

```diff
@@ -6,8 +6,11 @@ class Avo::Actions::Test::CloseModal < Avo::BaseAction
   }
 
   def handle(**args)
-    TestBuddy.hi "Hello from Avo::Actions::Test::CloseModal handle method"
     succeed "Modal closed!!"
     close_modal
+
+    append_to_response -> {
+      turbo_stream.set_title("Cool title")
+    }
```

Comment:
Haven't we settled on using an array notation?
I mean, why the extra block?

```suggestion
    append_to_response([
      turbo_stream.set_title("Cool title")
    ])
```

Reply:
I guess we could do 
```ruby
append_to_response do
  turbo_stream...
end
```

Reply:
Forget about it. A block is enough. I hope users don't mistake it for the Execution Context.

Reply:
LEt's also discuss this possibility


```ruby
    # turbo_stream_response will be executed instead the default `render turbo_stream: responses` from actions controller
    turbo_stream_response -> {
      render turbo_stream: [
        *default_responses, # can include or not the default responses
        turbo_stream.set_title("Cool title"),
        turbo_stream....
      ]
    }
```